docs/source/index.rst

```diff
@@ -1,6 +1,88 @@
-========================
-Scylla Monitoring Stack
-========================
+.. title:: Scylla Monitoring Stack
+
+.. raw:: html
+
+  <div class="landing landing--floating">
+
+
+.. hero-box::
+  :title: Scylla Monitoring Stack Documentation
+  :image: /_static/img/mascots/scylla-monitor.svg
+  :button_icon: fa fa-github
+  :button_url: https://github.com/scylladb/scylla-monitoring
+  :button_text: Scylla Monitoring Stack Project
+
+  .. include:: /_common/monitor-description.rst
+  Older versions of Scylla Monitoring Stack Documentation are available `here <https://docs.scylladb.com/operating-scylla/monitoring/>`_.
+
+.. raw:: html
+
+  <div class="landing__content landing__content">
+
+.. raw:: html
+
+  <div class="topics-grid grid-container full">
+
+  <div class="grid-x grid-margin-x">
+
+.. topic-box::
+    :title: Download and Install
+    :icon: fa fa-download
+    :link: /install/
```

Comment:
``` suggestion
    :link: install
```

@lauranovich Please do not add leading and trailing "/" (applies to all topic boxes in index.rst).